src/Controls/src/Core/PlatformConfiguration/iOSSpecific/Page.cs

```diff
@@ -161,29 +180,54 @@ public static Thickness SafeAreaInsets(this IPlatformElementConfiguration<iOS, F
 			return config;
 		}
 
-		/// <summary>Bindable property for <see cref="ModalPresentationStyle"/>.</summary>
+		/// <summary>
+		/// Defines the modal presentation style of the Page.
+		/// </summary>
 		public static readonly BindableProperty ModalPresentationStyleProperty =
 			BindableProperty.Create(nameof(ModalPresentationStyle), typeof(UIModalPresentationStyle), typeof(Page), UIModalPresentationStyle.FullScreen);
 
-		/// <include file="../../../../docs/Microsoft.Maui.Controls.PlatformConfiguration.iOSSpecific/Page.xml" path="//Member[@MemberName='ModalPresentationStyle']/Docs/*" />
+		/// <summary>
+		/// Gets the modal presentation style of the Page.
+		/// </summary>
+		/// <param name="config">The platform specific configuration that contains the element on which to perform the operation.</param>
+		/// <returns>The modal presentation style.</returns>
 		public static UIModalPresentationStyle ModalPresentationStyle(this IPlatformElementConfiguration<iOS, FormsElement> config)
 		{
 			return GetModalPresentationStyle(config.Element);
 		}
 
-		/// <include file="../../../../docs/Microsoft.Maui.Controls.PlatformConfiguration.iOSSpecific/Page.xml" path="//Member[@MemberName='SetModalPresentationStyle']/Docs/*" />
+		/// <summary>
+		/// Sets the modal presentation style.
+		/// </summary>
+		/// <param name="config">The platform specific configuration that contains the element on which to perform the operation.</param>
+		/// <param name="value">The modal presentation style.</param>
+		/// <returns>The platform specific configuration that contains the element on which to perform the operation.</returns>
 		public static IPlatformElementConfiguration<iOS, FormsElement> SetModalPresentationStyle(this IPlatformElementConfiguration<iOS, FormsElement> config, UIModalPresentationStyle value)
 		{
 			SetModalPresentationStyle(config.Element, value);
 			return config;
 		}
 
-		/// <include file="../../../../docs/Microsoft.Maui.Controls.PlatformConfiguration.iOSSpecific/Page.xml" path="//Member[@MemberName='GetModalPresentationStyle']/Docs/*" />
+		/// <summary>
+		/// Gets the current value of the UIModalPresentationStyle enumeration that's applied to the Page.
+		/// </summary>
+		/// <param name="element">A page, the VisualElement that occupies the entire screen.</param>
+		/// <returns>The current value of the UIModalPresentationStyle enumeration that's applied to the Page.</returns>
 		public static UIModalPresentationStyle GetModalPresentationStyle(BindableObject element)
 		{
 			return (UIModalPresentationStyle)element.GetValue(ModalPresentationStyleProperty);
 		}
 
+		/// <summary>
+		/// Sets the modal presentation style on a Page by specifying one of the following UIModalPresentationStyle enumeration values:
+		/// - FullScreen, which sets the modal presentation style to encompass the whole screen.By default, modal pages are displayed using this presentation style.
```

Comment:
I think that these definitions should go on `UIModalPresentationStyle` rather than here.